docs/UserGuide.adoc

```diff
@@ -665,6 +669,25 @@ To quickly revert address book data to the state of last use:
 ====
 // end::saveload[]
 
+
+== Non-command features
+
+// tag::interestCalculator[]
+=== Interest Calculator
+
+[NOTE]
+This is only applicable to debtors with a set interest rate. For the current version, only positive whole numbers are accepted. +
+
+[NOTE]
+`Codii` only checks through it's list of debtors when the user logs into the application. +
```

Comment:
logs into the application and updates all the `debt` and `totalDebt` fields for debtors who have a valid interest rate?

Reply:
its*